Source/WebCore/platform/graphics/cocoa/FontCacheCoreText.cpp

```diff
@@ -368,6 +368,37 @@ static inline float normalizeGXWeight(float value)
     return 523.7 * value - 109.3;
 }
 
+// These values were experimentally gathered from the various named weights of San Francisco.
+static struct {
+    float ctWeight;
+    float cssWeight;
+} keyframes[] = {
+    { -0.8, 30 },
+    { -0.4, 274 },
+    { 0, 400 },
+    { 0.23, 510 },
+    { 0.3, 590 },
+    { 0.4, 700 },
+    { 0.56, 860 },
+    { 0.62, 1000 },
+};
+static_assert(WTF_ARRAY_LENGTH(keyframes) > 0);
+
+float normalizeCTWeight(float value)
```

Comment:
"Normalize" and "denormalize" don't sound like the right terms to me. What about `convertCTWeightToCSSWeight` and vice versa?

Reply:
🤔 I use `normalize` in other places, under the assumption that we are a web engine so of course the "normal" form will be the CSS form. I think I take your point, though - I'll rename all the places in a follow-up patch.